### ISSUE TITLE
Set up transifex

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#2]
@MorrisJobke @LukasReschke could you please enable transifex for this repo? Thanks

**Comment by LukasReschke** [comment]
Assigning @MorrisJobke on this 😉

**Comment by MorrisJobke** [comment]
Yes. I will do this.

**Comment by ChristophWurst** [comment]
FYI: there's a little script that has to be run before the strings to translate can be processed – https://github.com/nextcloud/mail/blob/3408e7ad433cf600e20133042cc9ea683127d66b/translation-extractor.php

**Comment by MorrisJobke** [comment]
Done: 1c5c0151e5bf79045b5cc9bcc92802c67ada0a54

**Comment by MorrisJobke** [comment]
Weird ... it says it has read the file:

```
$ perl ./l10n.pl mail read
Mode: reading
  Processing mail
    Reading ./Gruntfile.js
    Reading ./appinfo/app.php
    Reading ./appinfo/application.php
...
    Reading ./tests/service/loggertest.php
    Reading ./translation-extractor.php
    Reading ./translations.js
```

But I can't see the content in transifex 😢

**Comment by MorrisJobke** [comment]
> But I can't see the content in transifex 😢

Fix is in #42

**Comment by MorrisJobke** [comment]
Setup is done and the first push by the bot already happened: 1c5c0151e5bf79045b5cc9bcc92802c67ada0a54

Closing
